README.md

```diff
@@ -12,16 +12,16 @@ stakeholders can be confident about the direction the project is evolving in.
 
 ## How to submit an RFC
 
-1. Copy `template.md` into `text/YYYY-MM-DD-my-feature.md`.
+1. Copy `template.md` into `text/PRID-my-feature.md`.
 2. Write the document and fill in the blanks.
 3. Submit a pull request.
 
 ## Timeline of an RFC
 
 1. An RFC is submitted as a PR.
 2. Discussion takes place, and the text is revised in response.
-3. The PR is merged or closed when at least two project maintainers reach
-   consensus.
+3. The PR is accepted or rejected when at least two project maintainers reach consensus.
```

Comment:
The previous language was chosen to be less confrontational

Reply:
With `4` now a tracking issue is required on accepted case so that we cannot interchange these words. In the other way, a proposal is rejected by technical reason is not offensive in my opinion.